pkg/cloud/cloud.go

```diff
@@ -542,6 +559,44 @@ func (c *cloud) GetSnapshotByName(ctx context.Context, name string) (snapshot *S
 	return c.ec2SnapshotResponseToStruct(ec2snapshot), nil
 }
 
+func (c *cloud) ListSnapshots(ctx context.Context, volumeID string, maxResults int64, nextToken string) (listSnapshotsResponse *ListSnapshotsResponse, err error) {
+	describeSnapshotsInput := &ec2.DescribeSnapshotsInput{}
+
+	if maxResults >= 5 {
```

Comment:
Since `controller.ListSnapshots` already has logic to guarantee `maxResults >= 5`, as `cloud` is an internal struct, I feel we can just set `describeSnapshotsInput.MaxResults = aws.Int64(maxResults)` without checking `if maxResults >= 5 {`.

Reply:
I just pushed another commit that enabled ListSnapshot sanity testing.  In order to satisfy the unit tests from the sanity testing packages, I had to push the logic down into the `cloud` package so controller was able to handle 0 < `MaxEntries` < 5.  Let me know how you want to proceed with this.  It's mostly cosmetic as we're just using `fakeCloudProvider` anyways.

Reply:
If that's what the sanity test checks, im okay with the change.